### DISCUSSION TITLE
Clipping content of ScrollViewer

### DISCUSSION THREAD

**Discussion by arivoir** [microsoft/microsoft-ui-xaml#7488] [Q&A]
How can I make the content of ScrollViewer to be clipped?

I'm using ScollViewer to create a data grid, where the column headers are rendered in the TopHeader.

![image](https://user-images.githubusercontent.com/17496741/114899259-31994080-9de9-11eb-9fe3-32dfd26622a3.png)

The Content renders behind the TopHeader.

I know there is the property CanContentRenderOutsideBounds, but it is false by default, and it looks it is not considering the TopHeader.

**Comment by StephenLPeters** [comment]
I think I would just make your header have an opaque background, or put the header outside of the scrollviewer (don't use the header property).

@RBrid FYI

**Comment by arivoir** [comment]
None of them is applicable, the opaque background doesn't let customers set their own background properly and it can not be out of the scroll-viewer because the headers also need to scroll horizontally.
